### ISSUE TITLE
conv4_3_norm is scaled

### ISSUE THREAD

**Issue by dimkastan** [weiliu89/caffe#735]
### Issue summary
conv4_3_norm performs the L2 normalization as described in ParseNet https://arxiv.org/pdf/1506.04579.pdf
However, I saw a difference between PyTorch (custom Matlab and custom Python) and Caffe-SSD.
More specifically, I noticed that the output is scaled by a constant factor 

### Steps to reproduce
First I loaded caffe-ssd in python, I loaded an image and I exported the conv4_3 and conv4_3_norm blobs into .mat objects.
Then I used the following code: <br />
    
    x      = torch.from_numpy(scio.loadmat('conv4_3.mat')['a'])
    x_norm = torch.from_numpy(scio.loadmat('conv4_3_norm.mat')['a'])
    norm = x.pow(2).sum(dim=1, keepdim=True).sqrt()+1E-10
    xmy_norm = 20*(x/norm )

which in my case needs to be scaled by 1.509

    xmy_norm/1.509
to return the correct result.

### Your system configuration
Operating system: Ubuntu 14.04
Compiler:               g++
CUDA version (if applicable): 8.0
CUDNN version (if applicable): 6.0
BLAS: openblas
Python or MATLAB version (for pycaffe and matcaffe respectively):
Python 2.7

**Comment by weiliu89** [comment]
Because the scale variable is changed after training.

**Comment by dimkastan** [comment]
Thanks for your reply.
 It seems that I am missing something. 
Isn't it defined by the following: 
layer {
  name: "conv4_3_norm"
  type: "Normalize"
  bottom: "conv4_3"
  top: "conv4_3_norm"
  norm_param {
    across_spatial: false
    scale_filler {
      type: "constant"
      value: 20
    }
    channel_shared: false
  }
Or during Test phase this value get overwritten somewhere or is there a case that Python wrapper alters this value during Normalization Layer parameter parsing?
